canonical/canonical.com
# Build the personal identity statement

## Done
Build the personal identity statement
Add a link to the new page on the Diversity page

## QA
- Open the demo and go to /careers/diversity
- Click the link to the statement
- Check the new "Personal identity at Canonical" page matches [the copy doc](https://docs.google.com/document/d/1C1xp9yIBtTutWEoyZnsttLQmslSMNuCPjDmabi9tx60/edit) 

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/4541
